sdcm/kafka/kafka_config.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+
+from pydantic import Field, BaseModel, Extra  # pylint: disable=no-name-in-module
+
+# pylint: disable=too-few-public-methods
+
+
+class ConnectorConfiguration(BaseModel):
+    # general options
+    connector_class: str = Field(alias="connector.class")
+    topics: Optional[str]
+
+    # scylla-cdc-source-connector options
+    # see https://github.com/scylladb/scylla-cdc-source-connector?tab=readme-ov-file#configuration
+    # and https://github.com/scylladb/scylla-cdc-source-connector?tab=readme-ov-file#advanced-administration
+    scylla_name: Optional[str] = Field(alias="scylla.name")
+    scylla_table_names: Optional[str] = Field(alias="scylla.table.names")
+    scylla_user: Optional[str] = Field(alias="scylla.user")
+    scylla_password: Optional[str] = Field(alias="scylla.password")
+
+    # kafka-connect-scylladb
+    # see https://github.com/scylladb/kafka-connect-scylladb/blob/master/documentation/CONFIG.md
+    scylladb_contact_points: Optional[str] = Field(alias="scylladb.contact.points")
+    scylladb_keyspace: Optional[str] = Field(alias="scylladb.keyspace")
+    scylladb_user: Optional[str] = Field(alias="scylladb.user")
+    scylladb_password: Optional[str] = Field(alias="scylladb.password")
```

Comment:
The diff between `scylladb_password` and `scylla_password` may be understood only looking at the source code and not by the attr name.
Better to rename it to reduce the ambiguity.

Reply:
Where do you see scylla_password here ?

Otherwise I don't understand what the question

Reply:
I see it on line 19. The same about the `scylla_user` and `scylladb_user`

Reply:
This configuration is an overlap of two different connectors

The discrepancy you see here is between them.

You can see in the link each one points to a different connector